assets/js/view/part/city.rb

```diff
@@ -61,13 +61,53 @@ def render_part
                   h(CitySlot, city: @city,
                               token: token,
                               slot_index: slot_index,
-                              radius: slot_radius,
+                              radius: SLOT_RADIUS,
                               reservation: @city.reservations[slot_index])
                 ])
             ])
         end
 
-        h(:g, { attrs: { class: 'city' } }, slots)
+        children = []
+
+        children << render_box(slots.size) if slots.size > 1
+
+        children += slots
+
+        h(:g, { attrs: { class: 'city' } }, children)
+      end
+
+      # TODOS:
+      # - do actual math and get points for the 3-slot hexagon, rather than
+      #   scaling the full-size hexagon
+      # - implement for 4, 5, and 6 slot cities
+      def render_box(slots)
+        box =
+          case slots
+          when 2
+            h(
+              :rect,
+              attrs: {
+                width: 2 * SLOT_RADIUS,
+                height: 2 * SLOT_RADIUS,
+                fill: 'white',
+                x: -SLOT_RADIUS,
+                y: -SLOT_RADIUS,
+              }
+            )
+          when 3
+            h(
+              :polygon,
+              attrs: {
+                points: '100,0 50,-87 -50,-87 -100,-0 -50,87 50,87',
+                fill: 'white',
+                transform: 'scale(0.458)',
+              }
+            )
+          end
+
+        h(:g, { attrs: { class: 'city-box' } }, [
```

Comment:
h('g.city_box') let's use all underscores for classes and ids

Reply:
9ad0115